Library/Homebrew/utils/gems.rb

```diff
@@ -145,6 +146,13 @@ def install_bundler_gems!(only_warn_on_failure: false, setup_path: true, groups:
     ENV["BUNDLE_WITH"] = groups.join(" ")
     ENV["BUNDLE_FROZEN"] = "true"
 
+    # System Ruby does not pick up the correct SDK by default.
+    if ENV["HOMEBREW_MACOS_SYSTEM_RUBY_NEW_ENOUGH"]
+      macos_major = ENV.fetch("HOMEBREW_MACOS_VERSION").partition(".").first
+      sdkroot = "/Library/Developer/CommandLineTools/SDKs/MacOSX#{macos_major}.sdk"
+      ENV["SDKROOT"] = sdkroot if Dir.exist?(sdkroot)
```

Comment:
Should we try falling back to something if that directory doesn't exist? Or, if we'd rather not do that, error out?

Reply:
We don't support installations without the CLT on non-latest macOS versions (unless you are doing a no-dev-tool flow, in which case you won't be able to install bundler gems anyway). `brew doctor` will flag the issue.

I guess it's maybe relevant to those using `HOMEBREW_BOOTSNAP`? Since they won't be able to even do `brew doctor`.